### PR TITLE
docs: every keyword

### PR DESCRIPTION
**Pull request by srenatus**
This:

- adds a brief not to the index page
- adds a section on "Every Keyword" to policy-language

Part of #2090.

### CODE REVIEW

File: docs/content/policy-language.md [hunk 1]
```diff
@@ -1124,24 +1125,43 @@ no_bitcoin_miners_using_negation with apps as [{"name": "bitcoin-miner"}, {"name
...
+    }
+}
+```
+
+{{< info >}}
+Whether you use negation, comprehensions, or `every` to express FOR ALL is up to you.
+The comprehension version is more concise and does not require a helper rule
+while the negation version is more verbose but a bit simpler and allows for
+more complex ORs.
+Ideally, the `every` variant allows you to express something very closely aligned
```
Comment (tsandall) [inline docs/content/policy-language.md:1162]: I'd think we want to encourage people to use `every` in most cases, no? I think we should be more prescriptive here.

File: docs/content/policy-language.md [hunk 2]
```diff
@@ -1303,6 +1323,61 @@ behaviour of other rules.
 For using the `some` keyword with iteration, see
 [the documentation of the `in` operator](#membership-and-iteration-in).
 
+## Every Keyword
+
+```live:eg/every0:module:merge_down
+import future.keywords.every
+
+p {
```
Comment (tsandall) [inline docs/content/policy-language.md:1331]: Can we reframe these examples in terms of the example data used throughout this doc?

File: docs/content/policy-language.md [hunk 3]
```diff
@@ -1303,6 +1323,61 @@ behaviour of other rules.
...
+
+Evaluating `every` does **not** introduce new bindings into the rule evaluation.
+
+Used with a key argument, the index, or property name (for objects), comes into the
+scope of the body evaluation:
+
+```live:eg/every1:module:merge_down
+import future.keywords.every
+
+p {
```
Comment (tsandall) [inline docs/content/policy-language.md:1362]: I'd split this example in two... one for arrays and one for objects. I guess we'd want a third example for sets.

File: docs/content/policy-language.md (hunk 1 above)
Comment (tsandall) [reply docs/content/policy-language.md:1162]: Perhaps we should mention that `every` has the same semantics as NOT-SOME-NOT rewriting?

**Review by tsandall — COMMENTED** [review]
Looks good! Let's make a few edits and then get this merged. 🎊 

Also, please update the EBNF in the Policy Reference.

**Comment by srenatus** [comment]
Well the last commit didn't do the trick. I'll get whatever is at odds sorted out soon.

**Review by tsandall — APPROVED** [review]
LGTM

**Pull request merged**